patches/master_patch.patch

```diff
@@ -1619,6 +1640,30 @@ index 9e0e5b96b8e38a6c29e601d091941dad28eec900..1463daf2462cfb3d8208a91f3fb3b0f1
  }
  
  void RenderWidgetTargeter::ViewWillBeDestroyed(RenderWidgetHostViewBase* view) {
+diff --git a/content/browser/storage_partition_impl_map.h b/content/browser/storage_partition_impl_map.h
+index ce0951bacd4dab08db92f6a87087279fcc7e4f09..10c7976815635c945079344e63ecf7f9f9932eaf 100644
+--- a/content/browser/storage_partition_impl_map.h
++++ b/content/browser/storage_partition_impl_map.h
+@@ -22,6 +22,10 @@ class FilePath;
+ class SequencedTaskRunner;
+ }  // namespace base
+ 
++namespace brave {
++class BraveBrowserContext;
++} // namespace brave
++
```

Comment:
please remove extra spaces here and below and I believe this can be put on one line